templates/docs/layouts/application.md

```diff
@@ -184,3 +184,21 @@ $main-panel-visible-width: 20rem;
 ```
 
 [View the full screen example of the application layout with custom JAAS panels](/docs/examples/layouts/application/JAAS/).
+
+## Import
+
+To import just this layout into your project, copy the snippet below and include it in your main Sass file.
+
+```scss
+// import Vanilla and include base mixins
+// this only needs to happen once in a given project
+@import 'vanilla-framework';
+@include vf-base;
+
+@include vf-l-application;
```

Comment:
I think it's just this one needed `vf-l-application` - it includes the `vf-l-application-panels` itself (in the end), and also uses the `--when-expanded` `--when-collapsed` internally. So including main mixin should be enough to have everything.

Reply:
I didn't spot that - thanks! removed